Open Judge System/Data/OJS.Data/Migrations/201708220830272_identity_update.cs

```diff
@@ -0,0 +1,56 @@
+namespace OJS.Data.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class identity_update : DbMigration
+    {
+        public override void Up()
+        {
+            RenameColumn(table: "dbo.AspNetUserClaims", name: "User_Id", newName: "UserId");
```

Comment:
All these rows must have `this.` in their beginning. We really must set up ur ReSharper... :)